lib/auth/clt.go

```diff
@@ -2002,6 +2002,41 @@ func (c *Client) DeleteAuthPreference(context.Context) error {
 	return trace.NotImplemented(notImplementedMessage)
 }
 
+// GetPAMConfig gets types.PAMConfig from the backend.
+func (c *Client) GetPAMConfig() (types.PAMConfig, error) {
+	out, err := c.Get(c.Endpoint("configuration", "pam"), url.Values{})
```

Comment:
Like @russjones wrote above - these are immediately obsolete and should only be available on the gRPC client?